.changeset/hot-schools-attack.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': major
+---
+
+Migrated the Popover component from Popper to Floating UI. Popper `modifiers` are longer supported, use `offsetProp` prop for flexible placement of floating element instead. `placement` prop can no longer have `auto*` values. Fixed `ProfileMenu` to use the updated `Popover` API. 
```

Comment:
It can be helpful to include links to further information for anyone who's curious.

The `ProfileMenu` changes are an implementation detail — since they don't impact developers, I wouldn't mention them.

```suggestion
Migrated the Popover component from [Popper](https://popper.js.org/) to [Floating UI](https://floating-ui.com/). Popper's `modifiers` are longer supported, use the `offset` prop for flexible placement of the floating element instead. The `placement` prop no longer accepts `auto*` values. 
```

Reply:
I'm a perfectionist 🙈